planning/autoware_obstacle_cruise_planner/include/autoware/obstacle_cruise_planner/common_structs.hpp

```diff
@@ -58,25 +58,42 @@ struct Obstacle
     const geometry_msgs::msg::Pose & arg_pose, const double ego_to_obstacle_distance,
     const double lat_dist_from_obstacle_to_traj)
   : stamp(arg_stamp),
+    ego_to_obstacle_distance(ego_to_obstacle_distance),
+    lat_dist_from_obstacle_to_traj(lat_dist_from_obstacle_to_traj),
```

Comment:
Can you remove these diff...? I'd like to reduce differential as much as possible.

Reply:
This is for suppressing "will be initialized after" warning. These 2 member variables are common to both PredictedObject and PointCloud, so I thought it would be better to move them above the other members for PredictedObject.

Reply:
OK. Thanks.